_includes/session_data.html

```diff
@@ -5,7 +5,11 @@
 <div class="popout">
   <section>
       <h1>{{talk.title}}</h1>
-      <h2>by {% for speaker in talk.speakers %}{{ site.data.speakers[speaker].name }} {% endfor %}</h2>
+      <h2>by
+        {% for speaker in talk.speakers %}
+        {{ site.data.speakers[speaker].name }}
+        {% unless forloop.last %}and{% endunless %}
```

Comment:
This will have a problem if we have three speakers `x and y and z` but it doesn't matter right now.

Reply:
I know, we don't have that in Paris so I didn't care to fix it yet.